src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
```

Comment:
Creating a new `Random` object every time `getRandomBall` is called is inefficient. Consider making `Random` a class-level field and initializing it once, then reusing it in this method. This will improve performance slightly by avoiding the overhead of creating a new `Random` object each time.